Account repo

Akongstad opened the pull request.

Akongstad committed:
- Setup repositories with unimplemented methods
- Begin preparing Infrastructure.tests

Akongstad committed:
Merge remote-tracking branch 'origin/main'

# Conflicts:
#	ProjectBank/Infrastructure.test/AccountRepositoryTests.cs
#	ProjectBank/Infrastructure.test/GlobalUsings.cs
#	ProjectBank/Infrastructure.test/KeywordRepositoryTests.cs
#	ProjectBank/Infrastructure.test/ProjectBank.Infrastructure.test.csproj
#	ProjectBank/Infrastructure.test/ProjectRepositoryTests.cs

Akongstad committed:
- AccountRepository.cs + tests
- Create (revisit)
- Read
- Readall

Stender98 reviewed (changes requested):
Den spytter 40 errors ud, så tænker at vente med merge til actions er gode 😅 Kan evt se på branchen inden mandag

Akongstad committed:
-Accountrepo 0.1 finished. CRUD implemented
- AccountRepositoryTests.cs with tests implemented for CRUD methods
- AccountRepositoryTests.cs with implements disposable pattern
